webextension/js/popup.js

```diff
@@ -101,6 +101,28 @@ let Logic = {
     }
     return this._currentIdentity;
   },
+
+  generateIdentityName() {
+    const defaultName = 'Container';
+    let ids = [];
+
+    // This loop populates the 'ids' array with all the already-used ids.
+    this._identities.forEach(identity => {
+      if (identity.name.startsWith(defaultName)) {
+        let id = parseInt(identity.name.substr(defaultName.length))
+        if (id) {
+          ids.push(id);
+        }
+      }
+    });
+
+    // Here we find the first valid id.
+    for (let id = 1;; ++id) {
+      if (ids.indexOf(id) == -1) {
```

Comment:
Should probably be `===`. In fact, shouldn't this have failed ESLint linting, per [/.eslintrc.js:9](https://github.com/mozilla/testpilot-containers/blob/3d5b58078db1ddc5635a3d46bf3a17399472ae5b/.eslintrc.js#L9)?